acct-creation-script/onboard.py

```diff
@@ -243,7 +247,7 @@ def get_assumed_session(environment, role):
 def get_access_templates(arpio_account, prod, recovery, arpio_auth_header):
     url = build_arpio_url('accounts', arpio_account, 'syncPairs',
                           prod[0], prod[1], recovery[0], recovery[1], 'accessTemplates')
-    body, code, _ = http_get(url, headers={arpio_auth_header})
+    body, code, _ = http_get(url, headers=dict(arpio_auth_header))
```

Comment:
If `arpio_auth_header` is already a dictionary, you don't need the `dict()` function call.  

In the old code, `{arpio_auth_header}` is actually using set-literal (not dict-literal) syntax, which was probably a bug at run-time (because `http_get` would have passed a set, not a dict, to `Request()` and maybe errored).

Reply:
Yes.